index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const cors = require('cors');
+const { pool } = require('./config');
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded());
```

Comment:
I'm not 100% sure, but do you need both of these currently? (json() and urlencoded())   My understanding is that `.json()` parses the body for POST requests while `urlencoded()` is a body parser for HTML post forms.  However, currently you don't have your BE setup for POSTS right now.